packages/wiz-ui-next/src/components/base/inputs/search-input/search-input.vue

```diff
@@ -1,6 +1,43 @@
 <template>
   <WizPopupContainer :expand="expand">
-    <div :class="styles.searchStyle">
+    <div
+      v-if="showSelectedItem && checkValues.length > 0"
+      :class="[
+        styles.searchInputSelectedItemStyle,
+        disabled && styles.searchInputDisabledStyle,
+        inputBorderStyle[state],
+      ]"
+      :style="{ width: computedInputWidth }"
+    >
+      <div :class="styles.searchInputInnerBoxStyle">
+        <WizHStack align="center" height="100%" gap="xs" pr="xl" :wrap="true">
+          <div v-for="item in checkValues" :key="item">
+            <span :class="styles.searchInputInnerBoxSelectedItemStyle">
+              <span :class="styles.searchInputInnerBoxSelectedLabelStyle">
+                {{ item }}
+              </span>
+              <button
+                type="button"
+                @click="onClear(item)"
+                @keypress.enter="onClear(item)"
+                @keydown="(e) => onBackspace(item, e)"
```

Comment:
`パラメーター 'e' の型は暗黙的に 'any' になります。`という警告を抑制するために型注釈が必要かもしれません
```suggestion
-               @keydown="(e) => onBackspace(item, e)"
+               @keydown="(e: KeyboardEvent) => onBackspace(item, e)"               
```